2023-02-24 13:30:06,727 INFO    MainThread:192272 [wandb_setup.py:_flush():69] setting env: {}
2023-02-24 13:30:06,727 INFO    MainThread:192272 [wandb_setup.py:_flush():69] setting login settings: {}
2023-02-24 13:30:06,728 INFO    MainThread:192272 [wandb_init.py:_log_setup():334] Logging user logs to /home/renfei/Documents/se3gcnn/wandb/run-20230224_133006-30uxazyk/logs/debug.log
2023-02-24 13:30:06,728 INFO    MainThread:192272 [wandb_init.py:_log_setup():335] Logging internal logs to /home/renfei/Documents/se3gcnn/wandb/run-20230224_133006-30uxazyk/logs/debug-internal.log
2023-02-24 13:30:06,728 INFO    MainThread:192272 [wandb_init.py:init():367] calling init triggers
2023-02-24 13:30:06,728 INFO    MainThread:192272 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2023-02-24 13:30:06,728 INFO    MainThread:192272 [wandb_init.py:init():416] starting backend
2023-02-24 13:30:06,728 INFO    MainThread:192272 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-02-24 13:30:06,736 INFO    MainThread:192272 [backend.py:ensure_launched():123] starting backend process...
2023-02-24 13:30:06,827 WARNING MainThread:192272 [wandb_init.py:init():763] interrupted
Traceback (most recent call last):
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 743, in init
    run = wi.init()
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/wandb_init.py", line 419, in init
    backend.ensure_launched()
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/site-packages/wandb/sdk/backend/backend.py", line 125, in ensure_launched
    self.wandb_process.start()
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/multiprocessing/context.py", line 284, in _Popen
    return Popen(process_obj)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/multiprocessing/popen_spawn_posix.py", line 32, in __init__
    super().__init__(process_obj)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/multiprocessing/popen_fork.py", line 19, in __init__
    self._launch(process_obj)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/multiprocessing/popen_spawn_posix.py", line 59, in _launch
    cmd, self._fds)
  File "/home/renfei/anaconda3/envs/torchenv/lib/python3.6/multiprocessing/util.py", line 417, in spawnv_passfds
    False, False, None)
KeyboardInterrupt
